awx/main/tasks/system.py

```diff
@@ -115,7 +114,9 @@ def dispatch_startup():
     m.reset_values()
 
     # Update Tower's rsyslog.conf file based on loggins settings in the db
-    reconfigure_rsyslog()
+    # Paylod is empty string because the configurer already has the data
+    with pg_bus_conn() as conn:
+        conn.notify('rsyslog_configurer', "")
```

Comment:
Do we even need to change anything about this call? This is the startup hook for the dispatcher. I see no reason to send it to a separate service in order to run it. Doing so creates new problems, like if the dispatcher starts up faster than the rsyslog configurer, then it might be listening and the message could get dropped.

Reply:
at the end of the day rsyslog wont be running in the same container as dispatcher so dispatcher wont have access to the rsyslog config file or the supervisord to restart rsyslog

running reconfigure_rsyslog in dispatcher will not work

Reply:
true, I need to revise this comment. We should continue the existing practice of restarting rsyslog on startup. But this should not be done by the dispatcher sending a message to the "rsyslog_configurer" channel. It can be done without race conditions by simply reconfiguring rsyslog on startup on the new service added here - the run_rsyslog_configurer service. So I'm still requesting that `reconfigure_rsyslog()` be moved from the dispatcher command startup to the new service startup (as opposed to task-ifying it)